azuki-core/src/main/kotlin/com/anaplan/engineering/azuki/core/runner/JUnitScenarioRunner.kt

```diff
@@ -370,19 +372,28 @@ class JUnitScenarioRunner<
             }
         }
         val nonParameterizedRuns = eacs + adapterTests + analysisScenarios + generatedScenarios + modellingExamples
-        return (if (parameterMethod == null) nonParameterizedRuns else parameterize(nonParameterizedRuns)).toMutableList()
+        return (if (parameterMethod == null) nonParameterizedRuns else parametrize(nonParameterizedRuns)).toMutableList()
     }
 
-    private fun parameterize(baseRuns: List<ScenarioRun<AF, CF, QF, AGF>>): List<ScenarioRun<AF, CF, QF, AGF>> {
+    private fun parametrize(baseRuns: List<ScenarioRun<AF, CF, QF, AGF>>): List<ScenarioRun<AF, CF, QF, AGF>> {
```

Comment:
:no_entry_sign: not only is 'parameterize' the dominant US spelling (https://en.wiktionary.org/wiki/parameterize), it is ubiquitous in CS